app/Services/PermissionService.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Http;
+
+class PermissionService
+{
+    public function checkPermissions(): bool
```

Comment:
```suggestion
    public function checkGitHubAppInstallation(): bool
```
More described what that function do.